tests/test_tpt_reports.py

```diff
@@ -170,3 +175,64 @@ def test_parse_json():
     assert payloads_meta["num_payloads"] == 1
     assert payloads_meta["payloads_blocked"] == 1
     assert payloads_meta["payloads_not_blocked"] == 1
+
+
+@patch("tpt_reports.tpt_reports.generate_reports")
+def test_no_output_directory(mock_generate_reports):
+    """Validate that no output directory argument uses default."""
+    patched_args = [
+        "bogus",
+        "--log-level=info",
+        "test",
+        "test",
+        "test",
+        TEST_JSON_FILE,
+    ]
+
+    # Patch usage arguments
+    with patch.object(sys, "argv", patched_args):
+        # Set mock return value
+        mock_generate_reports.return_value = True
+
+        # Call tpt_reports.main entry poifunction
```

Comment:
I think this is what you meant?
```suggestion
        # Call tpt_reports.main entry point function
```

Reply:
Yes. 😂 My brain stopped processing here apparently. Thanks!